src/lib/journey/stages/login.svelte

```diff
@@ -25,7 +25,7 @@
   import type { Maybe } from '$lib/interfaces';
   import CallbackMapper from '$journey/_utilities/callback-mapper.svelte';
 
-  // New API
+  export let componentStyle: 'app' | 'inline' | 'modal';
```

Comment:
app?

Reply:
This is for the future when this can be deployed as an app, rather than a widget.